### DISCUSSION TITLE
Postgresql Row Level Security

### DISCUSSION THREAD

**Discussion by okadts** [directus/directus#11624] [Archive]
Any help on how to implement Row Level Security with Postgresql and Directus?

Thank you

**Comment by rijkvanzanten** [comment]
[Permissions](https://docs.directus.io/configuration/users-roles-permissions/#users-roles-permissions) allow you to configure row level security 👍🏻

**Reply by okadts** [reply]
Any clue about the Row Level Security that built in the PostgreSQL itself?
I am testing Supabase that utilize this feature and wanted to combine with Directus on some module
Btw yes, the permissions on Directus is awesome 👍🏻👍🏻👍🏻
Thank you

**Reply by rijkvanzanten** [reply]
Directus itself doesn't manage postgres' row level security. That being said, if you configure it on the database directly, it'll still apply to the operations that directus executes.
